doc/using/Synthesis.rst

```diff
@@ -58,6 +58,37 @@ See :ghdlsharp:`1174` for on-going discussion about other output formats.
 .. TIP::
    Files can be provided in any order.
 
+.. _synthesis_options:
+
+Synthesis options
+==================
+
+In addition to analyze options, you can use following synthesis specific options:
```

Comment:
Since "additional" synthesis options are explained, might be worth adding a reference to "debugging options for synthesis": https://github.com/ghdl/ghdl/blob/8d5ae7bf63c616925306b42d42d01d1cebf3ffc7/src/ghdldrv/ghdlsynth.adb#L197-L214

Instead of documenting each of them, I'd just suggest users to find them in `src/ghdldrv/ghdlsynth.adb`.

Reply:
I added a hint with some explanation and a link to the ghdlsynth.adb file with name of the relevant procedure.

Reply:
There is a shortcut for referencing sources in the repo. See https://github.com/ghdl/ghdl/commit/1b23af06d9806867887aebad36b71c26e10584c1.

Reply:
Ups... https://github.com/ghdl/ghdl/commit/856c45e7f2374de4015c26cab5d02d092d53b267